upkeep.py

```diff
@@ -174,7 +174,16 @@ def get_upcoming_workdays(workdays_ahead: int = 2) -> typing.Iterator[datetime.d
     # to help ensure the most recent existing e-scrum issue is included even when other
     # non e-scrum issues exist
     issues = repo.get_issues(state='all', sort='number', direction='desc')
-    issues = issues[:min(10 + args.workdays_ahead, issues.totalCount)]
+    # If totalCount is working properly
```

Comment:
consider adding a link to the PR with the bug description or describing it in the comment